src/components/elements/input/NumberInput.tsx

```diff
@@ -85,7 +85,7 @@ const NumberInput = ({
 
   return (
     <TextInput
-      type='number'
+      type='text'
```

Comment:
turns off up/down arrow keys on the input field. It still functions as numeric from the input mode